mezzrow/oscar_apps/checkout/views.py

```diff
@@ -10,13 +10,22 @@
 
 BankcardForm = get_class('payment.forms', 'BankcardForm')
 
+from smallslive.users.models import SmallsUser
+
 
 class IndexView(views.IndexView):
     def form_valid(self, form):
         if form.is_guest_checkout():
             first_name = form.cleaned_data['first_name']
             last_name = form.cleaned_data['last_name']
             self.checkout_session.set_reservation_name(first_name, last_name)
+
+            # Capture this email address if it's not already in the database
+            a = SmallsUser.objects.get_or_create(
```

Comment:
Yes, it will. However, the user data created this way is easily identifiable against legacy accounts and should be deletable fairly easily.